hotsos/plugin_extensions/openstack/service_network_checks.py

```diff
@@ -73,9 +74,9 @@ def get_phy_port_health_info(self):
         etc) for any outliers detected.
         """
         port_health_info = {}
-        for project in ['nova', 'neutron', 'octavia']:
-            _project = getattr(self, project)
-            if _project and _project.bind_interfaces:
+        for project in [f.name for f in fields(self.project_helpers)]:
+            _project = getattr(self.project_helpers, project)
+            if _project.bind_interfaces:
```

Comment:
_project can still be invalid object, old check is still necessary even after using fileds